src/app/not-found.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from 'next/link';
+
+export default function NotFound() {
+  return (
+    <div className="min-h-screen bg-rose-50 flex items-center justify-center p-4">
+      <div className="max-w-md w-full text-center">
+        <h1 className="text-6xl font-bold text-gray-800 mb-4">404</h1>
+        <h2 className="text-2xl font-semibold text-gray-700 mb-6">Sayfa Bulunamadı</h2>
+        <p className="text-gray-600 mb-8">
+        Aradığınız sayfa mevcut değil veya taşınmış olabilir. Lütfen ana sayfaya dönmeyi deneyin.
+        </p>
+        <Link 
+          href="/" 
+          className="text-teal-600 hover:text-teal-700 font-medium text-lg"
```

Comment:
In the other place, teal-800 is used for hover state. I suggest updating the hover color to 800 here as well. Let's use 700 instead of 600 as well to make it aligned with other places in the app.